server/src/dtos/user.dto.ts

```diff
@@ -104,17 +131,12 @@ export class UpdateUserDto {
   quotaSizeInBytes?: number | null;
 }
 
-export class UserDto {
-  id!: string;
-  name!: string;
-  email!: string;
-  profileImagePath!: string;
-  @IsEnum(UserAvatarColor)
-  @ApiProperty({ enumName: 'UserAvatarColor', enum: UserAvatarColor })
-  avatarColor!: UserAvatarColor;
+export class UserAdminDeleteDto {
+  @ValidateBoolean({ optional: true })
+  force?: boolean;
 }
 
-export class UserResponseDto extends UserDto {
+export class UserAdminResponseDto extends UserResponseDto {
```

Comment:
Yeah, the benefit is minimal. I had in mind the instances which have several admins who are not necessarily all system administrators

Reply:
We don't even allow adding admins through the GUI :P
Maybe that is something we could look at in the future, but for now it's probably fine, yea

Reply:
It's been a while since I've seen people ask how to do this, but I'm sure there were instructions on discord on how to give admin privileges to other users (by directly changing the value in db lol). But, yes, that's clearly not an issue right now